modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/FileWriteAheadLogManager.java

```diff
@@ -1856,25 +1735,7 @@ private void init(SegmentAware segmentAware) throws IgniteCheckedException {
          * @return The absolute indices of min and max archived files.
          */
         private IgniteBiTuple<Long, Long> scanMinMaxArchiveIndices() throws IgniteCheckedException {
-            TreeMap<Long, FileDescriptor> archiveIndices = new TreeMap<>();
```

Comment:
Let's keep this method here and move `ft.walArchive().listFiles(WAL_SEGMENT_COMPACTED_OR_RAW_FILE_FILTER)` to `NodeFileTree`.